tools/test-proxy/Azure.Sdk.Tools.TestProxy.Tests/TransformTests.cs

```diff
@@ -13,20 +13,26 @@ public class TransformTests
         [Fact]
         public async Task ConditionalTransformAppliesForRegex()
         {
-            var clientIdTransform = new StorageRequestIdTransform(condition: new ApplyCondition() { UriRegex = @".+/Tables.*" });
+            var clientIdTransform = new HeaderTransform("x-ms-client-request-id", condition: new ApplyCondition
+            {
+                RequestHeaderCondition = new HeaderCondition
+                {
+                    Key = "x-ms-client-request-id"
+                },
+                UriRegex = @".+/Tables.*"
+            });
+
             RecordingHandler testRecordingHandler = new RecordingHandler(Directory.GetCurrentDirectory());
             testRecordingHandler.Transforms.Clear();
             testRecordingHandler.Transforms.Add(clientIdTransform);
 
             var playbackContext = new DefaultHttpContext();
-            var transformedContent = "This should still show up.";
```

Comment:
This had to be removed because transforms are now applied over the EntryRecord so the in-memory change doesn't get reflected in the loaded record session file.